### PR TITLE
Add node descriptions

### PR DESCRIPTION
**Pull request by PhilipMusk**
[Text extracts of node descriptions - updated and realigned - v1 back into model.docx](https://github.com/CBIIT/icdc-model-tool/files/7809389/Text.extracts.of.node.descriptions.-.updated.and.realigned.-.v1.back.into.model.docx)

### CODE REVIEW

**Comment by majensen** [comment]
IMO the descriptions are too wordy with a lot of information about how the database is set up. That's not for the model to describe. 
Here are some more compact descriptions.

[Text extracts of node descriptions - updated and realigned - maj.docx](https://github.com/CBIIT/icdc-model-tool/files/7831447/Text.extracts.of.node.descriptions.-.updated.and.realigned.-.maj.docx)

**Comment by PhilipMusk** [comment]
With review input noted, merged in the node descriptions as originally added to the model definition, and will revise at the appropriate time. There are several advantages to taking this approach. Not least of which is the fact that we can safely merge these non-breaking changes, which are invisible to ICDC's current Production code, into the the master copy of the model definitions, upstream of it's planned branching into a pre-production branch. Within which breaking changes, such as the change to "Enum:", can now be tested.
